lib/literal/array.rb

```diff
@@ -196,6 +218,24 @@ def reject!(...)
 		self
 	end
 
+	def replace(value)
+		value = case value
+						when Array
+							value
+						when Literal::Array
+							value.to_a
```

Comment:
I don’t know if `Array#replace(value)` will mutate the given value or just use it as a reference. If it doesn’t mutate, we could call `.__value__` here instead of `.to_a`.

Reply:
I don't  _think_ value gets mutated but not 100% sure the best way to check
<img width="494" alt="CleanShot 2024-11-11 at 15 52 31@2x" src="https://github.com/user-attachments/assets/5dc01a6d-84ea-445c-8e29-24a6883a2cc5">